virttest/qemu_vm.py

```diff
@@ -1493,6 +1493,17 @@ def sort_key(dev):
                     cmdline = add_qxl(qxl_dev_nr, addr)
                     devices.insert(StrDev('qxl', cmdline=cmdline))
 
+        # Add watchdog device
+        if params.get("enable_watchdog", "no") == "yes":
+            parent_bus = None
+            watchdog_device_type = params.get("watchdog_device_type", None)
+            watchdog_action = params.get("watchdog_action", "reset")
+            cmd = add_watchdog(devices, watchdog_device_type, watchdog_action)
+            if watchdog_device_type == 'i6300esb':
```

Comment:
Did you try `devices.is_pci_device(watchdog_device_type)`?

Reply:
Same with the comment above.  it can work.  However, I am not confident to determine `-device i6300esb` is same with `-watchdog i6300esb`

Reply:
@yanan-fu Emm, both okay to me.